data.rs

```diff
@@ -0,0 +1,170 @@
+use crate::PSP22Error;
+use ink::{
+    prelude::{vec, vec::Vec},
+    primitives::AccountId,
+    storage::Mapping,
+};
+
+pub enum PSP22Event {
+    Transfer(Option<AccountId>, Option<AccountId>, u128),
```

Comment:
```suggestion
    Transfer { 
        from: Option<AccountId>, 
        to: Option<AccountId>, 
        amount: u128
        },
```
This is a user-facing type that they're supposed to map into a proper `PSP22Event` in their contracts. We don't want them to mess up the fields.

Reply:
Good idea, done